modules/module-list.nix

```diff
@@ -19,6 +20,9 @@
   ./system/defaults/smb.nix
   ./system/defaults/SoftwareUpdate.nix
   ./system/defaults/spaces.nix
+  ./system/defaults/textedit.nix
+  # timemachine is not writable
+  #./system/defaults/timemachine.nix
```

Comment:
What's the reason for adding this if it can't be configured?

Reply:
I was hoping, that somebody might know an answer to why it's not writable.
If nobody knows, I'll remove the lines and the file.
I guess we could leave the list conversion in, could be useful for other properties